core/src/test/scala/org/apache/spark/PartitioningSuite.scala

```diff
@@ -244,6 +244,10 @@ class PartitioningSuite extends SparkFunSuite with SharedSparkContext with Priva
     assert(abs(6.0/2 - rdd.mean) < 0.01)
     assert(abs(1.0 - rdd.variance) < 0.01)
     assert(abs(1.0 - rdd.stdev) < 0.01)
+    assert(abs(1.0 - rdd.popVariance) < 0.01)
```

Comment:
Perhaps assert that variance == popVariance instead and likewise for stdev? because really we need them to be identical. Otherwise LGTM